### PR TITLE
Set the tooltip to work on the invalid icon

### PR DESCRIPTION
**Pull request by GittyBlau**

### CODE REVIEW

File: Spectrum/src/com/unhurdle/spectrum/TextField.as [hunk 1]
```diff
@@ -228,7 +228,11 @@ package com.unhurdle.spectrum
       }
       invalidTooltip = new AdaptiveTooltipBead();
       invalidTooltip.flavor = "negative";
-      addBead(invalidTooltip);
+      invalidIcon.addBead(invalidTooltip);
+      invalidIcon.setStyle("pointer-events","auto");
+      COMPILE::JS{
+            invalidIcon.width = invalidIcon.element.clientWidth;
```
Comment (Harbs) [inline Spectrum/src/com/unhurdle/spectrum/TextField.as:234]: Why is this necessary?

Comment (GittyBlau) [reply Spectrum/src/com/unhurdle/spectrum/TextField.as:234]: For the position of the tooltip. determinePosition() returns pt.x =NaN (=0) because the width of the icon is undefined,
